flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartition.java

```diff
@@ -434,15 +445,12 @@ public int getBuffersInBacklog() {
 		}
 	}
 
-	private boolean shouldNotifyDataAvailable() {
-		// Notify only when we added first finished buffer.
-		return readView != null && !flushRequested && getNumberOfFinishedBuffers() == 1;
+	private boolean mayNotifyDataAvailable() {
```

Comment:
`mayNotifyDataAvailable` seems fit for `void` return type. I prefer to previous `shouldNotifyDataAvailable` which more fits for `boolean` return type.